### PR TITLE
Libdocs

### PR DESCRIPTION
**Pull request by IhabBendidi**
documentation for library

### CODE REVIEW

File: ihub/README.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+##### Output
+{'category': 'Unknown', 'box': (576, 1093, 1242, 428), 'precision': 1}
+
+
+```json
+
+[{
+  "category" : "bendidi_ihab",
+  "precision" : "0.9",
+  "box" : (576, 1093, 1242, 428)
```
Comment (egor-jerome) [inline ihub/README.md]: json doesn't have tuple type, this should be an array instead : [576, 1093....]

File: ihub/README.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+```python
+
+import zihub.ihub.predict as predict
+response = predict.recognize_face(imagePath)
+
+```
+
+##### Output
+{'category': 'Unknown', 'box': (576, 1093, 1242, 428), 'precision': 1}
```
Comment (egor-jerome) [inline ihub/README.md]: is `precision` a float (1.0) ?

File: ihub/README.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+
+```json
+
+[{
+  "category" : "bendidi_ihab",
+  "precision" : "0.9",
```
Comment (egor-jerome) [inline ihub/README.md]: is `precision` a string or a float?

File: ihub/README.md (hunk 1 above)
Comment (egor-jerome) [reply ihub/README.md]: ... if the output is in json format of course

File: ihub/README.md (hunk 2 above)
Comment (IhabBendidi) [reply ihub/README.md]: Actually that line I just forgot there, it was supposed to be deleted. Will do now

File: ihub/README.md (hunk 3 above)
Comment (IhabBendidi) [reply ihub/README.md]: Float, my bad

**Pull request merged**